packages/moonstone/VideoPlayer/VideoPlayer.module.less

```diff
@@ -424,9 +424,7 @@
 
 		.overlay {
 			&.high-contrast-scrim::after {
-				:global(.enact-a11y-high-contrast) & {
-					background: @moon-video-player-high-contrast-scrim;
-				}
+				background: linear-gradient(fade(@moon-video-player-high-contrast-scrim-gradient-color, 0%) 0%, fade(@moon-video-player-high-contrast-scrim-gradient-color, 60%) 100%);
```

Comment:
What if this was reconfigured to be like the following:
```
.overlay {
    &::after {
        background: @moon-video-player-overlay-scrim-bg;
    }
}
```
Then in `colors.less`:
```
@moon-video-player-overlay-scrim-bg: initial; // or `transparent none` to affect both bg-color and bg-image.
```
And in `colors-highcontrast.less`:
```
@moon-video-player-overlay-scrim-bg: linear-gradient(fade(@moon-black, 0%) 0%, fade(@moon-black, 60%) 100%);
```
We may not need the `&.high-contrast-scrim` class at all since we can simply _always_ have it and just render it as blank/empty under normal circumstances.

This approach would leverage our new high-contrast code, and remove some old implementation that's now obsolete.

Reply:
👍 Sounds good. Just updated the scrim.